main-app/app/scripts/services/game-model.js

```diff
@@ -19,7 +19,21 @@
                     function (data, status) {
                         alert('Server Error:' + status + ' information ' + data);
                     });
-            };
+
+            if (this.player2 === 'human') {return currentPlayer == player2;
```

Comment:
If I remember correctly, player one was starting, after that I would be clicking another square and I would get the error ‘Player cannot play again’ since this new code has been implemented, the currentPlayer is undefined, which seems to be an issue with the if statement - after taking it out correctly all conditions where human or computer is player 1 works.

On 20 Jan 2015, at 09:42, Keith Barrow <notifications@github.com<mailto:notifications@github.com>> wrote:


In main-app/app/scripts/services/game-model.js<https://github.com/SalamanderMan/NoughtsAndCrosses/pull/23#discussion-diff-23211321>:

> @@ -19,7 +19,21 @@
>                      function (data, status) {
>                          alert('Server Error:' + status + ' information ' + data);
>                      });
> -            };
> +
> +            if (this.player2 === 'human') {return currentPlayer == player2;


What happens if you select two human players?
Who actually makes the first move.

—
Reply to this email directly or view it on GitHub<https://github.com/SalamanderMan/NoughtsAndCrosses/pull/23/files#r23211321>.